superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -134,18 +136,8 @@ def get_recipe_text(self, distributor, license_text, die_msg=None):
             self.license = self.license.replace(' ', '-')
             ret += 'LICENSE = "' + self.license + '"\n'
         elif isinstance(self.license, list):
-            self.license = self.license[0].replace(' ', '-')
-            ret += 'LICENSE = "' + self.license + '"\n'
-            """
-            TODO(allenh1): add this functionality
-            first = True
-            for lic in self.license:
-                if not first:
-                    ret += ' '
-                    first = False
-                ret += lic
+            ret += 'LICENSE = "' + ' & '.join(self.license)
```

Comment:
This is very close. Make sure to call the `getlicense` function on each entry of the list! Something like the following would probably do the trick.

```py
' & '.join([getlicense(l) for l in self.license])
```

Reply:
OH yeah, that makes sense, cause the hackyness is gone so every item needs to be taken into account im on it :+1: